requirements-ci.txt

```diff
@@ -8,7 +8,7 @@ autograd
 toml
 appdirs
 semantic_version
-autoray
+autoray==0.6.8
```

Comment:
```suggestion
autoray>=0.6.1,<0.6.10
```

Reply:
We will be adding a combined changelog in the feature branch.